.template-lintrc.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = {
+  extends: 'recommended'
+};
```

Comment:
do we have any templates here that require this file?

Reply:
removed